package-lock.json

```diff
@@ -5539,8 +5539,7 @@
 						"ansi-regex": {
 							"version": "2.1.1",
 							"bundled": true,
-							"dev": true,
-							"optional": true
```

Comment:
@nfmohit-wpmudev I'm a bit concerned about merging this branch with these changes to the package-lock.json file. If I remember correctly, I think they're created if you run a different version of npm to other people. (I'm on 6.9.0)

Would you be able to undo the changes?

Something like this should do the trick:
```
git pull --rebase origin master
git checkout master package-lock.json
git push --force-with-lease
```

Reply:
Whoops! That was unintended, apologies. I've reverted the changes on [this commit](https://github.com/WordPress/gutenberg/pull/10128/commits/5ee5208fb48db44245882e84e268b40e139b1d2c) and updated the NPM version. Thank you very much for letting me know and the instructions (as always) ❤️